From 12aba2e99c7eb831e817ca00a9dc0f1aa8cb8d5f Mon Sep 17 00:00:00 2001
From: Shreya Tode <shreya.todei@intel.com>
Date: Mon, 26 Jul 2021 11:01:59 +0530
Subject: [PATCH] Fix for CTS Media Perf Test Cases

Change perf range in XML for TGL

Tracked on: OAM-97895
Signed-off-by: Shreya Tode <shreya.tode@intel.com>

diff --git a/groups/codecs/configurable/media_codecs_performance_tgl.xml b/groups/codecs/configurable/media_codecs_performance_tgl.xml
index 28cc892..94ac048 100644
--- a/groups/codecs/configurable/media_codecs_performance_tgl.xml
+++ b/groups/codecs/configurable/media_codecs_performance_tgl.xml
@@ -31,7 +31,7 @@
             <Limit name="measured-frame-rate-640x360" range="200-220" />
             <Limit name="measured-frame-rate-720x480" range="170-190" />
             <Limit name="measured-frame-rate-1280x720" range="70-120" />
-            <Limit name="measured-frame-rate-1920x1080" range="40-70" />
+            <Limit name="measured-frame-rate-1920x1080" range="159-240" />
         </MediaCodec>
         <MediaCodec name="c2.android.mpeg4.decoder" type="video/mp4v-es" update="true">
             <Limit name="measured-frame-rate-176x144" range="600-1100" />
@@ -39,8 +39,8 @@
         <MediaCodec name="c2.android.vp8.decoder" type="video/x-vnd.on2.vp8" update="true">
             <Limit name="measured-frame-rate-320x180" range="860-930" />
             <Limit name="measured-frame-rate-640x360" range="290-320" />
-            <Limit name="measured-frame-rate-1280x720" range="65-70" />
-            <Limit name="measured-frame-rate-1920x1080" range="25-30" />
+            <Limit name="measured-frame-rate-1280x720" range="140-214" />
+            <Limit name="measured-frame-rate-1920x1080" range="66-96" />
         </MediaCodec>
         <MediaCodec name="c2.android.vp9.decoder" type="video/x-vnd.on2.vp9" update="true">
             <Limit name="measured-frame-rate-320x180" range="530-600" />
@@ -49,31 +49,31 @@
             <Limit name="measured-frame-rate-1920x1080" range="50-100" />
         </MediaCodec>
         <MediaCodec name="OMX.Intel.hw_vd.h264" type="video/avc" update="true">
-            <Limit name="measured-frame-rate-320x240" range="1500-2000" />
+            <Limit name="measured-frame-rate-320x240" range="3864-6146" />
             <Limit name="measured-frame-rate-720x480" range="1000-2000" />
-            <Limit name="measured-frame-rate-1280x720" range="460-900" />
+            <Limit name="measured-frame-rate-1280x720" range="1585-2378" />
             <Limit name="measured-frame-rate-1920x1088" range="320-390" />
         </MediaCodec>
         <MediaCodec name="OMX.Intel.hw_vd.h265" type="video/hevc" update="true">
-            <Limit name="measured-frame-rate-352x288" range="1200-1250" />
+            <Limit name="measured-frame-rate-352x288" range="2943-4507" />
             <Limit name="measured-frame-rate-640x360" range="1000-1400" />
-            <Limit name="measured-frame-rate-720x480" range="820-1020" />
-            <Limit name="measured-frame-rate-1280x720" range="540-580" />
+            <Limit name="measured-frame-rate-720x480" range="2259-3394" />
+            <Limit name="measured-frame-rate-1280x720" range="1454-2230" />
             <Limit name="measured-frame-rate-1920x1080" range="430-470" />
-            <Limit name="measured-frame-rate-3840x2160" range="120-130" />
+            <Limit name="measured-frame-rate-3840x2160" range="295-471" />
         </MediaCodec>
         <MediaCodec name="OMX.Intel.hw_vd.vp8" type="video/x-vnd.on2.vp8" update="true">
-            <Limit name="measured-frame-rate-320x180" range="700-1450" />
-            <Limit name="measured-frame-rate-640x360" range="400-950" />
-            <Limit name="measured-frame-rate-1280x720" range="150-600" />
+            <Limit name="measured-frame-rate-320x180" range="2941-5290" />
+            <Limit name="measured-frame-rate-640x360" range="2344-3410" />
+            <Limit name="measured-frame-rate-1280x720" range="1184-1720" />
             <Limit name="measured-frame-rate-1920x1080" range="100-330" />
         </MediaCodec>
         <MediaCodec name="OMX.Intel.hw_vd.vp9" type="video/x-vnd.on2.vp9" update="true">
-            <Limit name="measured-frame-rate-320x180" range="1300-1800" />
-            <Limit name="measured-frame-rate-640x360" range="500-1300" />
-            <Limit name="measured-frame-rate-1280x720" range="560-590" />
-            <Limit name="measured-frame-rate-1920x1080" range="350-420" />
-            <Limit name="measured-frame-rate-3840x2160" range="100-130" />
+            <Limit name="measured-frame-rate-320x180" range="3374-4844" />
+            <Limit name="measured-frame-rate-640x360" range="2948-4515" />
+            <Limit name="measured-frame-rate-1280x720" range="1576-2248" />
+            <Limit name="measured-frame-rate-1920x1080" range="1128-1662" />
+            <Limit name="measured-frame-rate-3840x2160" range="327-480" />
         </MediaCodec>
     </Decoders>
     <Encoders>
@@ -105,16 +105,16 @@
         </MediaCodec>
         <MediaCodec name="OMX.Intel.hw_ve.h264" type="video/avc" update="true">
             <Limit name="measured-frame-rate-320x240" range="260-500" />
-            <Limit name="measured-frame-rate-720x480" range="140-160" />
+            <Limit name="measured-frame-rate-720x480" range="454-685" />
             <Limit name="measured-frame-rate-1280x720" range="75-110" />
             <Limit name="measured-frame-rate-1920x1080" range="50-55" />
         </MediaCodec>
         <MediaCodec name="OMX.Intel.hw_ve.h265" type="video/hevc" update="true">
             <Limit name="measured-frame-rate-320x240" range="200-400" />
-            <Limit name="measured-frame-rate-720x480" range="120-160" />
-            <Limit name="measured-frame-rate-1280x720" range="70-90" />
-            <Limit name="measured-frame-rate-1920x1080" range="35-45" />
-            <Limit name="measured-frame-rate-3840x2160" range="10-14" />
+            <Limit name="measured-frame-rate-720x480" range="394-606" />
+            <Limit name="measured-frame-rate-1280x720" range="204-306" />
+            <Limit name="measured-frame-rate-1920x1080" range="98-147" />
+            <Limit name="measured-frame-rate-3840x2160" range="28-42" />
         </MediaCodec>
     </Encoders>
 </MediaCodecs>
-- 
2.32.0

